app/models/user.rb

```diff
@@ -0,0 +1,12 @@
+class User < ApplicationRecord
+  has_many :highlights, dependent: :destroy
+  has_many :user_sources, dependent: :destroy
+
+  def increment_num_highlights(by: 1)
+    update(num_highlights: num_highlights + by)
+  end
+
+  def increment_num_annotation_characters(by: 1)
```

Comment:
no default here? (`1` is a reasonable default for `increment_num_highlights` but not as an expected annotation length)